### DISCUSSION TITLE
Microsoft Azure student sign in issue

### DISCUSSION THREAD

**Discussion by swati1012** [github/india#501] [Students]
I am trying to sign in using the email id provided by the college but I am getting
"Your email domain is not currently registered with us. You can choose another verification method."
What should I do next?

**Comment by DevendraPhaniKumar** [comment]
I am facing the issue too.. It is difficult to create a student account unless the school is registered by Microsoft. Please guide us on it @ey-mentor1 @ey-mentor2 @EY-Mentor3 @EY-Mentor4

**Comment by meet-bariya** [comment]
I guess you are using microsoft account to sign into azure. Sign out from current session and while login use github account instead of microsoft account
